### PR TITLE
Small fixes on theme

### PR DESCRIPTION
**Pull request by XinYenFon**
Signed-off-by: Antes antes@simplemachines.org

### CODE REVIEW

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -2794,7 +2798,7 @@ dl.addrules dt.floatleft {
 	background: #f0f4f7;
 }
 .windowbg:nth-of-type(odd) {
-	background: #fdfdfd; /* needs new color */
+	background: #EEEEEE;
```
Comment (live627) [inline Themes/default/css/index.css]: Clashes with theme.

File: Themes/default/css/index.css [hunk 2]
```diff
@@ -1431,8 +1433,10 @@ ul li.greeting {
 	background-position: 1px -210px;
 	padding: 0 4px 0 22px;
 }
-.quickbuttons li a.unapprove_button {
+.quickbuttons li a.unapprove_button,
+.quickbuttons li a.unread_button {
 	background-position: 2px -238px;
+	padding: 0 4px 0 22px;
```
Comment (live627) [inline Themes/default/css/index.css]: Knocks bottom button out of alignment IMO.
fireshot screen capture 169 - automated test 632 - localhost_smf2_1_index_php_topic 635_msg30071 new

File: Themes/default/css/index.css (hunk 1 above)
Comment (XinYenFon) [inline Themes/default/css/index.css]: I really really bad at finding good color...

Comment (live627) [inline Themes/default/css/index.css]: Wasn't the old one fine?

Comment (XinYenFon) [inline Themes/default/css/index.css]: Too whiteish? I don't know, I asked for opinions in topic (private board), but none said anything. Let me revert back, we can discuss it later on (just a one line).

**Pull request merged**